### PR TITLE
wlroots: Update to v0.18.1

### PR DESCRIPTION
**Pull request by ReillyBrogan**
Changes:
- Update wlroots to v0.18.1
- Update sway to v1.10 and sync with Arch
- Add libsfdo at v0.1.3 for labwc
- Update labwc to v0.8.1

### CODE REVIEW

**Comment by ReillyBrogan** [comment]
For post-sync

File: packages/s/sway/files/50-systemd-user.conf [hunk 1]
```diff
@@ -2,4 +2,15 @@
...
 
-exec systemctl --user import-environment DISPLAY WAYLAND_DISPLAY SWAYSOCK XDG_CURRENT_DESKTOP
+# Upstream refuses to set XDG_CURRENT_DESKTOP so we have to.
+exec systemctl --user set-environment XDG_CURRENT_DESKTOP=sway
+exec systemctl --user import-environment DISPLAY \
+                                         SWAYSOCK \
+                                         WAYLAND_DISPLAY \
+                                         XDG_CURRENT_DESKTOP
+
+exec hash dbus-update-activation-environment 2>/dev/null && \
```
Comment (ermo) [inline packages/s/sway/files/50-systemd-user.conf:12]: Would probably be kind to leave a message here explaining why the `hash` statement is necessary for the sake of future maintainers?

**Review by ermo — DISMISSED** [review]
Just a small question re. why we're calling the `hash` thing, otherwise LGTM.

File: packages/s/sway/files/50-systemd-user.conf (hunk 1 above)
Comment (ReillyBrogan) [reply packages/s/sway/files/50-systemd-user.conf:12]: TBH I'm not actually sure, I just copied that from the corresponding Arch package. We can look into it once this lands.

**Comment by ReillyBrogan** [comment]
CC @davidjharder and @Facundo-c-c as Sway and labwc maintainers respectively.

**Comment by davidjharder** [comment]
Thanks Reilly, new sway release requires this.

**Pull request merged**